centrifuge/src/main/java/io/github/centrifugal/centrifuge/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * File: package-info.java
+ * Make all method parameters @NonNull by default
+ */
+@ParametersAreNonnullByDefault
```

Comment:
How does adding this file change the behaviour of SDK?

Reply:
This is equivalent of adding `@NotNull` to parameters of all methods in the package, which can be overridden by `@ Nullable` in each specific case.